### ISSUE TITLE
FoE Helper - GB Investment Not Working (Version 3.8.0.0)

### ISSUE THREAD

**Issue by tpaciorek** [mainIine/foe-helfer-extension#2984]
**Description:**  
In the latest version of FoE Helper (3.8.0.0), the "Budowa Pereł Architektury" (Construction of Architectural Pearls) feature does not display any content. The window appears, but it remains blank, as shown in the screenshot. I have tried restarting the game and reinstalling the extension, but the issue persists. Could you please investigate and resolve this issue?

**Steps to Reproduce:**
1. Open the Forge of Empires game with the FoE Helper extension enabled.
2. Navigate to the "Budowa Pereł Architektury" feature.
3. Observe the blank screen.

**Expected Result:**  
The feature should display relevant information and allow interaction for managing architectural pearl construction.

**Actual Result:**  
The window opens but remains completely empty.

**Version:**  
FoE Helper v3.8.0.0

**Screenshot:**  
(Attach the screenshot showing the issue)

Thank you for your assistance!

![1111test](https://github.com/user-attachments/assets/c8fb6cdb-8f00-4ff8-b542-21bd75e14667)

**Comment by Beelzebob6666** [comment]
yes, there currently is an issue with that - not sure, if the feature will return yet

**Comment by tpaciorek** [comment]
That's too bad because that feature was so cool. :(

**Comment by outoftheline** [comment]
can you explain a bit more what was so helpful about it?

**Comment by tpaciorek** [comment]
The module for Forge of Empires that supported the development of Great Buildings (GBs) was incredibly helpful because it allowed me to optimize the upgrading process. It showed which GB could be upgraded at the lowest cost while yielding the most benefits. Thanks to this feature, I was able to upgrade Great Buildings in a balanced way, maximizing the return in Forge Points relative to the amount spent. This was the main function I relied on, as it helped me manage resources efficiently and get the best possible development gains.
